docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -385,6 +385,35 @@ The following sections provide information about how ELSER performs on different
 hardwares and compares the model performance to {es} BM25 and other strong 
 baselines such as Splade or OpenAI.
 
+[discrete]
+[[version-overview]]
+=== Version overview
+
+The ELSER model has two main versions: ELSER V2 and ELSER V1.
+ELSER V2 has a **platform specific** version that is designed to run on Linux 
```

Comment:
(optional) I think it would be good to mention that V2 ". . . is *only* designed to run on Linux . . ." just as this is mentioned below.